### ISSUE TITLE
Build a Serverless Web App on CockroachCloud

### ISSUE THREAD

**Issue by jseldess** [cockroachdb/docs#8977]
We've discussed writing a tutorial for building a simple serverless app on CC. 

Some tutorials to serve as inspiration:
- https://aws.amazon.com/getting-started/hands-on/build-serverless-web-app-lambda-apigateway-s3-dynamodb-cognito/
- https://dev.to/abiodunjames/learn-serverless-on-gcp-by-building-a-simple-calendar-api-549c
- https://cloud.google.com/solutions/serverless-web-performance-monitoring-using-cloud-functions

Good reference of serverless options on GCP: https://cloud.google.com/serverless-options
More reference: https://medium.com/serverlessguru/aws-to-gcp-web-applications-89ed92070832

**Comment by jseldess** [comment]
@ericharmeling, @vy-ton, is this something we want to do in Q4, or is Q1 more realistic?

**Comment by ericharmeling** [comment]
> @ericharmeling, @vy-ton, is this something we want to do in Q4, or is Q1 more realistic?

I think Q1 is a little more realistic. It would be a nice asset to align with the 21.1 release.

**Comment by ericharmeling** [comment]
Moving this to the backlog for now. I don't think this is in scope for the release at this point.

**Comment by jseldess** [comment]
Agreed, @ericharmeling. In fact, we might end up just building a Cockroach U course on this as a next step. cc @dweatherbee, since this ticket has some potentially helpful resources.
